### PR TITLE
instructions for other developers

### PR DESCRIPTION
**Pull request by kevinoliveira**
improvments made on readme to help other developers get started, instructions were added on how to run the development server, run a api server alongside, run tests, check for type errors and linting

### CODE REVIEW

File: playground/README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
 # @sdkgen/playground
 
+The default web playground for sdkgen APIs, allowing easy interaction with deployed or local servers.
+
+# How to use
+
+just open a web browser on https://yourApiEndpoint.com/playground
```
Comment (lbguilherme) [inline playground/README.md:7]: Capitalize Just

File: playground/README.md [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+npm i 
+```
+
+## Running locally
+```
+npm run dev
+```
+a local version of the playground will be avaliable on http://localhost:4545/playground/ 
+
+by the default the playground will make requests to http://localhost:8000/, it's recomended to run an local sdkgen server alongside the playground dev server, here is an example of [very simple sdkgen server](https://github.com/kevinoliveira/sdkgen-helloworld) that you can use to develop.
```
Comment (lbguilherme) [inline playground/README.md:32]: By default

File: playground/README.md (hunk 1 above)
Comment (kevinoliveira) [reply playground/README.md]: fixed

File: playground/README.md (hunk 2 above)
Comment (kevinoliveira) [reply playground/README.md]: fixed

**Comment by lbguilherme** [comment]
I'm closing this for now. We should move forward to more docs at sdkgen.github.io in the future.

**Pull request closed without merging**